### ISSUE TITLE
Commented out comments can produce infinite formatting. 

### ISSUE THREAD

**Issue by dalemyers** [nicklockwood/SwiftFormat#609]
Here's an example file (notice that everything is commented out):

```swift
/*
class CheckComments {

 /*
    * Some badly indented comment
    *
    * Some lines
    * Go Here
  */
  func doSomething() {
    // Pass
  }
}
*/
```

You can see that there is a badly indented comment above the `doSomething` function. When swiftformat is run (Just out of the box with `swiftformat /path/to/file` is enough to repro), then those indented lines in that indented comment get further indented by one space every time swiftformat is run. The end result is that using a flag like `--dryrun` or `--lint` _always_ states that there are errors. 

Here's some example output of running with the `--lint` flag:

```
Running SwiftFormat...
(lint mode - no files will be changed.)
/Users/dalemy/Desktop/comments.swift:5:1: warning: (indent) Indent code in accordance with the scope level.
/Users/dalemy/Desktop/comments.swift:6:1: warning: (indent) Indent code in accordance with the scope level.
/Users/dalemy/Desktop/comments.swift:7:1: warning: (indent) Indent code in accordance with the scope level.
/Users/dalemy/Desktop/comments.swift:8:1: warning: (indent) Indent code in accordance with the scope level.
warning: No swift version was specified, so some formatting features were disabled. Specify the version of swift you are using with the --swiftversion command line option, or by adding a .swift-version file to your project.
SwiftFormat completed in 0.07s.
Source input did not pass lint check.
1/1 files require formatting.
```

**Comment by nicklockwood** [comment]
Thanks for reporting. I'll try to get this fixed asap. In the meantime you can work around it in this case with:

```swift
// swiftformat:disable indent
/*
class CheckComments {

 /*
    * Some badly indented comment
    *
    * Some lines
    * Go Here
  */
  func doSomething() {
    // Pass
  }
}
*/
// swiftformat:enable indent
```

**Comment by 1oo7** [comment]
I also noticed this one. Weird bug.

**Comment by nicklockwood** [comment]
Fixed in 0.44.8
